models/experimental/grok/demo/demo.py

```diff
@@ -0,0 +1,313 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+import os
+import torch
+import json
+import tt_lib as ttl
+import pytest
+from loguru import logger
+from time import time
+
+# Set Grok flags for CI, if CI environment is setup
+if os.getenv("CI") == "true":
```

Comment:
https://github.com/tenstorrent/tt-metal/pull/10371/files
we have a utility function recently merged for checking if "CI")

Reply:
That function cannot be used in our code. The purpose of these checks are to set environment variables that are necessary before ttnn is imported. However, conftest.py has a complex import chain that ends up importing ttnn to implement is_grayskull. Even if that were resolved, relying on nobody in any of those upstream modules adding a ttnn import in the future would be... wildly hopeful.